This test tries to walk a sysfs path and hits a null pointer exception.

/tmp/guix-build-ceph-12.0.0.drv-0/ceph-12.0.0/src/test/common/test_blkdev.cc:32: Failure
Expected: (dir) != (nullptr), actual: NULL vs 8-byte object <00-00 00-00 00-00 00-00>

diff --git a/src/test/common/CMakeLists.txt b/src/test/common/CMakeLists.txt
--- a/src/test/common/CMakeLists.txt
+++ b/src/test/common/CMakeLists.txt
@@ -12,15 +12,6 @@
   ${CMAKE_DL_LIBS}
   )
 
-if(HAVE_BLKID)
-  # unittest_blkdev
-  add_executable(unittest_blkdev
-    test_blkdev.cc
-    )
-  add_ceph_unittest(unittest_blkdev)
-  target_link_libraries(unittest_blkdev ceph-common ${BLKID_LIBRARIES})
-endif()
-
 # unittest_bloom_filter
 add_executable(unittest_bloom_filter
   test_bloom_filter.cc
